scripts/ci/check_pull_request.py

```diff
@@ -83,7 +83,10 @@ def check_pull_request(title, body):
         error_flag = regex_line(title)
         for line in body:
             if line.startswith('['):
-                error_flag = regex_line(line) or error_flag
+                # If edit history notes, ignore title check result
+                ref = re.findall(r'[\[](.*?)[\]]', line)
+                if ref and ref[0] not in ['Component Name 1', 'Component Name 2']:
+                    error_flag = regex_line(line)
```

Comment:
It seems the logic is not so correct, for example, if there is no error in the last line, even if there are error in other lines, the `error_flag` will be `False`, thus no error is reported.

Reply:
![image](https://user-images.githubusercontent.com/18628534/158775972-04f50718-1c6b-4ac5-988c-6d7e63dab9dc.png)